src/hotspot/share/opto/phasetype.hpp

```diff
@@ -186,13 +195,10 @@ class PhaseNameValidator {
     }
   }
 
-  bool is_valid() const {
-    return _valid;
-  }
-
-  const char* what() const {
-    return _bad;
-  }
+  const BitMap& mask() const { return _mask; }
+  bool is_valid() const { return _valid; }
+  bool is_mask_set() const { return _set; }
```

Comment:
Why not use `_mask.count_one_bits()`? Then you can drop the `_set` field.

Reply:
True, fixed now.